Cargo.toml

```diff
@@ -7,8 +7,6 @@ authors = ["The New Relic Agent Control Team"]
 publish = false
 license-file = "./LICENSE.md"
 
-[lints.rust]
-missing_docs = "warn"
```

Comment:
"I will return..."

Reply:
oh i actually didn't want to commit this , i had it like that for local dev, but anyway i think we agreed to not force this until we have proper docs to avoid the endless warnings